spec/outputs/dynatrace_spec.rb

```diff
@@ -186,6 +186,17 @@
         expect(subject).to have_received(:send_event).exactly(3).times
       end
     end
+
+    context 'with more than 4.5MB of events' do
```

Comment:
Please also add a test for handling:

- one event < 4.5 MB followed by an event > 4.5 MB
- one event > 4.5 MB followed by an event < 4.5 MB

You could also consider adding a third test that shows that configuring `max_payload_size` to a different value has the expected effect.

Reply:
Would the last suggestion - a test for the config itself - be easy to implement?

Reply:
maybe. i'll give it a go